packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -410,106 +570,655 @@ class _CupertinoAlertActionSection extends StatelessWidget {
   // don't have many actions.
   final ScrollController scrollController;
 
-  bool get _layoutActionsVertically => children.length > 2;
+  @override
+  _CupertinoAlertActionSectionState createState() {
+    return new _CupertinoAlertActionSectionState();
+  }
+}
+
+class _CupertinoAlertActionSectionState extends State<_CupertinoAlertActionSection> {
+  static _CupertinoAlertActionSectionState of(BuildContext context) {
+    return context.ancestorStateOfType(
+      const TypeMatcher<_CupertinoAlertActionSectionState>(),
+    );
+  }
+
+  final Set<int> _pressedButtons = new Set<int>();
+
+  void onButtonDown(int index) {
```

Comment:
When naming callbacks, use FooCallback for the typedef, onFoo for the callback argument or property, and handleFoo for the method that is called.

Reply:
Renamed those 2 methods to start with 'handle...'